hotspot/src/os/linux/vm/os_linux.cpp

```diff
@@ -1949,6 +1949,9 @@ void * os::dll_load(const char *filename, char *ebuf, int ebuflen)
   #ifndef EM_AARCH64
   #define EM_AARCH64    183               /* ARM AARCH64 */
   #endif
+  #ifndef EM_RISCV
+  #define EM_RISCV      243               /* RISC-V */
+  #endif
```

Comment:
It will be more consistent if you put this after definition of `EM_LOONGARCH` like the other places.

Reply:
Thanks for the review! Fixed.